rest-dbus.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
```

Comment:
This brings up an interesting discussion perhaps we should have.  It is reasonable for the daemons to provide a sample systemd file, but are we always going to use this in openbmc?  I assume we may likely want to customize the systemd for each daemon in openbmc.  We would do this in the distro or in the application?